opm/output/eclipse/EclipseIO.cpp

```diff
@@ -337,6 +337,25 @@ void EclipseIO::Impl::writeINITFile( const data::Solution& simProps, const NNC&
     }
 
 
+    //Write Integer Vector Map
+    {
+        std::map<std::string, std::vector<int> >::iterator it = map.begin();
```

Comment:
Use range based for loops instead of naked iterators:
```C++
for( const auto& pair : map)  {
     const std::string& key = pair.first;
     const std::vector<int>& int_data = pair.second;
     ...
     if (int_data.size() == this->grid.getCartesianSize( ))
        writeKeyword( fortio , key , int_data );
     else  {
         std::vector<int> global_copy = grid.scatterVector( int_data );
         writeKeyword( fortio , key , global_copy );
     }
}
```